models/Order.js

```diff
@@ -7,25 +7,47 @@ const orderSchema = new mongoose.Schema({
   status: {
     type: String,
     required: true,
-    enum: ['sent', 'pending', 'canceled'],
-    default: 'pending',
+    enum: ['CREATED', 'CONFIRMED', 'CANCELLED', 'REFUNDED', 'SENT', 'DELIVERED'],
+    default: 'CREATED',
   },
   user: {
     firstName: String,
     lastName: String,
+    patronymicName: String,
     email: String,
     phoneNumber: Number,
-    address: Address,
   },
   dateOfCreation: {
     type: Date,
     default: Date.now,
   },
+  lastUpdatedDate: Date,
+  completed: Boolean,
+  userComment: {
+    type: String,
+    default: '',
+  },
+  adminComment: {
+    type: String,
+    default: '',
+  },
+  cancellationReason: {
+    type: String,
+    default: '',
+  },
   delivery: {
     sentOn: Date,
     sentBy: String,
+    courier: Boolean,
+    courierOffice: Number,
     invoiceNumber: String,
+    serviceType: {
+      type: String,
+      enum: ['WarehouseWarehouse', 'WarehouseDoors'],
```

Comment:
Please, make it uppercased with underscores, as for `status`, let's follow the same formatting for all enum values

Reply:
I was provided with these values by Petro - they are char for char the ones Nova Poshta uses

Reply:
Field removed